exercise2.js

```diff
@@ -0,0 +1,13 @@
+var numeros = [1, 2, 3, 4, 5];
+class ArrayFunctions {
+    static printSpecial(array) {
+        return array.join(" --- ");
+    };
+
+ static doubleArray(array) {
+  return array.map(number => number * 2);
+    }
+    static superPower(array) {
+      return array.reduce( (sum, number, i) => sum + (number * (Math.pow(10, i))))
+    }
+}
```

Comment:
the first 2 exercises are the same, try to work on the 2 exercises left if you have time